Cubical/Structures/QuotientRing.agda

```diff
@@ -0,0 +1,248 @@
+{-# OPTIONS --cubical --safe #-}
+
+module Cubical.Structures.QuotientRing where
+
+open import Cubical.Foundations.Prelude
+open import Cubical.Foundations.Equiv
+open import Cubical.Foundations.HLevels
+open import Cubical.HITs.SetQuotients.Base
+open import Cubical.HITs.SetQuotients.Properties
+open import Cubical.Structures.Ring
+open import Cubical.Structures.Ideal
+
+
+private
+  variable
+    ℓ : Level
+
+module _ (R′ : Ring {ℓ}) (I : ⟨ R′ ⟩  → hProp ℓ) (I-isIdeal : isIdeal R′ I) where
+  private
+    open ring-syntax R′
+    open isIdeal I-isIdeal
+    open ring-axioms R′
+    open calculations R′
+    R = ⟨ R′ ⟩
+
+  R/I : Type ℓ
+  R/I = R / (λ x y → x - y ∈ I)
+
+  private
+    -isDistributive : (x y : R) →  (- x) + (- y) ≡ - (x + y)
```

Comment:
Shouldn't this be a general lemma about rings and not a private one?

Reply:
Yes, I'll move it.